steamworks/__init__.py

```
@@ -0,0 +1,217 @@
+from steamworks.interfaces.workshop     import SteamWorkshop
+from steamworks.interfaces.microtxn     import SteamMicroTxn
+from steamworks.interfaces.input        import SteamInput
+
+os.environ['LD_LIBRARY_PATH'] = os.getcwd()
+
+
+class STEAMWORKS(object):
+    """
+        Primary STEAMWORKS class used for fundamental handling of the STEAMWORKS API
+    """
+    _arch = Arch.x64  # 直接使用64位架构，避免导入问题
```

Ruff (0.14.5)

`Arch` may be undefined, or defined from star imports

(F405)